core/src/main/scala/JwtClaim.scala

```diff
@@ -75,47 +78,47 @@ class JwtClaim(
     JwtClaim(content, issuer, subject, audience, expiration, notBefore, issuedAt, Option(id))
   }
 
-  def expiresIn(seconds: Long): JwtClaim = {
+  def expiresIn(seconds: Long)(implicit clock: Clock): JwtClaim = {
```

Comment:
Went back and forth on whether a `JwtClaim` should have the clock in the constructor or on the method params. Ended up going with the latter since users may want to parse `JwtClaims` without doing any time based checks on them.